client/landing/stepper/declarative-flow/internals/steps-repository/newsletter-setup/index.tsx

```diff
@@ -1,18 +1,14 @@
 import { isEnabled } from '@automattic/calypso-config';
```

Comment:
Since we now handle all logic in the paid membership components, most of the changes in this file just involve removing that logic here.